prody/proteins/header.py

```diff
@@ -1041,7 +1041,11 @@ def buildBiomolecules(header, atoms, biomol=None):
 
     biomt = header.get('biomoltrans')
     if not isinstance(biomt, dict) or len(biomt) == 0:
-        raise ValueError("header doesn't contain biomolecular transformations")
+        if header.get('experiment') != 'X-RAY DIFFRACTION':
+            LOGGER.warn("This is not an X-ray structure so no transformation was applied")
+        else:
+            raise ValueError("header doesn't contain biomolecular transformations")
+            return None
```

Comment:
There is no need to return None if an error is already raised.

Reply:
Oh yeah

Reply:
Now fixed